home.html

```diff
@@ -0,0 +1,224 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="./style.css" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato&display=swap"
+      rel="stylesheet"
+    />
+    <script
+      src="https://kit.fontawesome.com/00fad0ddbb.js"
+      crossorigin="anonymous"
+    ></script>
+  </head>
+  <body>
+    <header>
+      <div class="social-media">
+        <div></div>
+        <div class="social-media-links">
+          <i class="fa-brands logo-icon fa-square-facebook"></i>
+          <i class="fa-brands logo-icon fa-twitter"></i>
+          <a href="">English</a>
+          <a href="">My Page</a>
+          <a href="">Logout</a>
+        </div>
+      </div>
+
+      <nav class="navbar nav-mobile disappearing-nav1">
+        <div class="hamburger">
+          <div id="myHamburger" class="ham-container hamburger">
+            <div class="bar1"></div>
+            <div class="bar2"></div>
+            <div class="bar3"></div>
+          </div>
+          <div id="myDropdown" class="dropdown-content">
+            <a href="./home.html" class="nav-link">Home</a>
+            <a href="./about.html" class="nav-link">About</a>
+            <a href="#Contact" class="nav-link">Venue</a>
+          </div>
+        </div>
+      </nav>
+
+      <nav class="menu-bar">
+        <div class="topnav">
+          <div class="bx1">
+            <a href="home.html" class="logo-txt"
+              >F<i class="fa-brands fa-google"></i> MusicFest</a
+            >
+          </div>
+          <div class="bx2">
+            <a href="">Home</a>
+            <a href="about.html">About</a>
+            <a href="#">Venue</a>
+          </div>
+        </div>
+      </nav>
+    </header>
+    <main>
+      <section class="first-sec">
+        <div class="first-sec-container">
+          <div class="first-sec-heading">
+            <h3>"Be a Part Musical History"</h3>
+            <h1>
+              Festus <i class="fa-brands fa-google">roup</i> Music Festival
+            </h1>
+          </div>
+          <div class="description">
+            <p>
+              The biggest music event in decades. Attended by more than 100
+              musicians from all over the world. Get your ticket now!
+            </p>
+          </div>
+          <div class="date">
+            <h2>2022.11.26(Sat.)</h2>
+            <p>@ Accra International Conference Center, Accra-Ghana</p>
+          </div>
+        </div>
+      </section>
+
+      <section class="second-sec">
```

Comment:
- [ ] Kindly check on your design for this section, your divider color is different from the Behance design, also your articles should be five and not four, below is a comparison for better understanding:

Behance Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/61408860/181730411-1657a002-41a2-45c9-b0e9-b755ae516830.png) |  ![image](https://user-images.githubusercontent.com/61408860/181730441-c621f8b3-b47e-4963-8d41-377e1bf8d864.png)